GLaDOS.py

```diff
@@ -116,22 +116,18 @@
     print("Client logged in as {}, in the following guild : {}"
           "".format(bot.user.name, bot.guild.name))
 
-
-# Handle errors
-# Taken from
-# https://github.com/916253/Kurisu/blob/31b1b747e0d839181162114a6e5731a3c58ee34f/run.py#L88
 @bot.event
 async def on_command_error(ctx, error):
     if isinstance(error, commands.errors.CommandNotFound):
         pass
     if isinstance(error, commands.errors.CheckFailure):
+        # Discussion: ignore failed check

```

Comment:
This is a discussion that will affect all commands.

Should we, instead of showing a error message, just ignore it when people use commands they were never suppose to